src/sentry/workflow_engine/endpoints/organization_available_action_index.py

```diff
@@ -0,0 +1,138 @@
+from collections import defaultdict
+
+from drf_spectacular.utils import extend_schema
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.integrations.services.integration import RpcIntegration
+from sentry.sentry_apps.services.app import app_service
+from sentry.workflow_engine.endpoints.serializers import (
+    ActionHandlerSerializer,
+    ActionHandlerSerializerResponse,
+)
+from sentry.workflow_engine.models import Action
+from sentry.workflow_engine.processors.action import (
+    get_available_action_integrations_for_org,
+    get_notification_plugins_for_org,
+)
+from sentry.workflow_engine.registry import action_handler_registry
+
+
+@region_silo_endpoint
+class OrganizationAvailableActionIndexEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ISSUES
+
```

Comment:
should this have a permission class?

Reply:
I don't think so? I think anyone who has access to the org (has `org:read`) should be able to call this endpoint, and this is handled by inheriting `OrganizationEndpoint`